src/components/ui/StylesWrapperButton.vue

```diff
@@ -0,0 +1,248 @@
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator';
+import SameLinkBtnProps from './SameLinkBtnProps.vue';
+import ArrowIcon from '../icons/ArrowIcon.vue';
+import LoaderIcon from '../icons/LoaderIcon.vue';
+
+@Component({
+  components: {
+    ArrowIcon,
+    LoaderIcon,
+  },
+})
+class StylesWrapperButton extends SameLinkBtnProps {
+  get loaderFill() {
+    return this.type === 'secondary' ||
+      (this.tag === 'link' && this.type !== 'black')
+      ? 'url(#paint0_angular_4802:1871)'
+      : 'url(#paint0_angular_4802:187)';
+  }
+
+  get arrowIconClassName() {
+    return 'styles-wrapper-button__icon styles-wrapper-button__icon_fill';
+  }
+  get className() {
+    return `styles-wrapper-button styles-wrapper-button_size_${this.size} styles-wrapper-button_type_${this.type} styles-wrapper-button_border-radius-size_${this.border}`;
+  }
```

Comment:
get classList() {
    return [
      'base-button',
      `base-button_${this.type}`,
      this.hasDefaultSlot ? 'base-button_not-only-icon' : 'base-button_only-icon',
    ]
  }

Reply:
make class like this